syntax = "proto3";

package yml;

service YmlPostMaker{
  rpc UploadPosts (UploadRequest) returns (UploadResponse);
}

message UploadRequest{
  int64 from_user = 1;
  string ymlUrl = 2;
}

message UploadResponse{
  int32 code = 1;
  string state = 2;
}